share/wake/lib/core/json.wake

```diff
@@ -58,12 +58,11 @@ export def parseJSONBody (body: String): Result JValue Error =
     Fail cause = Fail (makeError cause)
 
 export def parseJSONFile (path: Path): Result JValue Error =
-  def imp f = prim "json_file"
-  match (getPathResult path)
-    Pass f = match (imp f.getPathName)
-      Pass body = Pass body
-      Fail f = Fail (makeError f)
-    Fail f = Fail f
+    def imp f =
+        prim "json_file"
+    match (imp path.getPathName)
+        Pass body = Pass body
+        Fail f = Fail (makeError f)
```

Comment:
I think the standard library still needs to defensively use `getPathResult`.

Reply:
Makes sense.  I'll restore those.